libantworld/agent.h

```diff
@@ -73,8 +75,8 @@ class AntAgent
     }
 
 private:
-    Vector3<degree_t> m_Attitude{ 0_deg, 0_deg, 0_deg };
-    Vector3<meter_t> m_Position{ 0_m, 0_m, 0_m };
+    Vector3<degree_t> m_Attitude{{ 0_deg, 0_deg, 0_deg }};
+    Vector3<meter_t> m_Position{{ 0_m, 0_m, 0_m }};
```

Comment:
Do the extra set of curlies do anything? I'm guessing it's calling the array constructor with an array argument rather than passing the args directly, but does that change things?

Reply:
It's required for more elderly compilers